src/shell/notification.js

```diff
@@ -378,28 +367,7 @@ export function unpatchGtkNotificationDaemon() {
 const _addNotification = NotificationDaemon.GtkNotificationDaemonAppSource.prototype.addNotification;
 
 export function patchGtkNotificationSources() {
-    // This should diverge as little as possible from the original
-    // eslint-disable-next-line func-style
-    const addNotification = function (notificationId, notificationParams, showBanner) {
-        this._notificationPending = true;
-
-        if (this._notifications[notificationId])
-            this._notifications[notificationId].destroy(MessageTray.NotificationDestroyedReason.REPLACED);
-
-        const notification = this._createNotification(notificationParams);
-        notification.connect('destroy', (notification, reason) => {
-            this._withdrawGSConnectNotification(notification, reason);
-            delete this._notifications[notificationId];
-        });
-        this._notifications[notificationId] = notification;
-
-        if (showBanner)
-            this.showNotification(notification);
-        else
-            this.pushNotification(notification);
-
-        this._notificationPending = false;
-    };
+    const addNotification = Source.prototype.addNotification;
 
```

Comment:
This can also be removed, since it's no-op now.

Reply:
Indeed, it's the legacy of my first debugging hours :smile: Just removed it with the latest commit :+1: